Character.cs

```diff
@@ -755,5 +761,19 @@ internal static void ProcessSetTitleMessage(NetworkMessageEventArgs e)
                 Logging.LogError(ex);
             }
         }
+
+        internal static string GetLocation()
+        {
+            Decal.Adapter.Wrappers.CharacterFilter cf = MyCore.CharacterFilter;
+            var coords = MyCore.WorldFilter.GetByName(cf.Name).First.Coordinates();
+
+            if ( coords == null ) // Indoors?
+                return "";
```

Comment:
Assumption here. I've noticed (as a player) that coordinates don't really exist indoors. I also saw `StatsDump` being defensive -- so I assume if there are no coordinates that they must be indoors?

Could be useful to observe what's possible from `Coordinates()` to remove or accept the defensiveness.

Reply:
Right, coordinates are only an outdoors thing. Indoors isn't even Euclidian IIRC. However, we can always know whether the player is outdoors or indoors and, if the player is indoors, we can actually tell what dungeon they're in (which I'd like store).

Another of my plugins adapts code from GoArrow,

- How it's called: https://github.com/amoeba/TownCrier/blob/7da759035b6751096e2fd31991968b5e69c3ba99/Utilities.cs#L303
- The class backing the call: https://github.com/amoeba/TownCrier/blob/main/Coordinate.cs

I'll merge your work and then add support for reporting dungeons from there.